CMakeLists.txt

```diff
@@ -18,7 +18,9 @@ option(ONNX_COVERAGE "Build with coverage instrumentation" OFF)
 option(ONNX_BUILD_TESTS "Build ONNX C++ APIs Tests" OFF)
 
 # Set C++11 as standard for the whole project
-set(CMAKE_CXX_STANDARD 11)
+IF(NOT MSVC)
```

Comment:
Doesn't MSVC support c++11?
@raymondxyang @linkerzhang

Reply:
@bddppq The flag is useless in MSVC. Starting from C++14, it can be controlled by some flags like `/c++:14`. But for c++ 11, it can be just determined by MSVC version and is enabled by default.

Reply:
@peterjc123   Then by default it would be?

Reply:
I don't think onnx can support VC verions earlier than 2015. According to https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version, after your this change, it will default to C++14. Am I right?